rest/src/discovery-rest.h

```diff
@@ -9,11 +9,11 @@ static constexpr const char* AgentName = "discovery-ng_rest";
 inline fty::disco::Message message(const std::string& subj)
 {
     fty::disco::Message msg;
-    msg.meta.to      = "discovery-ng";
+    msg.meta.to      = "fty-discovery-ng";
```

Comment:
are you 100% sure that all other agents / rest will query the right name, and not just disco-ng?

Reply:
Yes, only the RESt client request the fty-discovery-ng service
(I've made a global IPM2 search on all repos...)